CMakeLists.txt

```diff
@@ -109,38 +110,23 @@ add_custom_target(check-all)
 add_custom_target(docs ALL)
 add_dependencies(docs mlir-doc)
 
-option(AIR_BUILD_RUNTIME "Build the air runtime" ON)
-if (NOT LibXAIE_FOUND)
-	message(WARNING "LibXAIE not found, AIR runtime will not be built.")
-	set(AIR_BUILD_RUNTIME OFF)
-endif()
-
-if(CMAKE_USE_TOOLCHAIN_AIRHOST)
-	set(AIRHOST_TOOLCHAIN_OPT ${CMAKE_TOOLCHAIN_FILE_OPT})
-else()
-	set(AIRHOST_TOOLCHAIN_OPT ${CMAKE_TOOLCHAIN_FILE})
-endif() 
-
-if(AIR_BUILD_RUNTIME)
-	message("AIRHOST_TOOLCHAIN_OPT=${AIRHOST_TOOLCHAIN_OPT}")
-	message("Sysroot=${Sysroot}")
-	ExternalProject_Add(air_runtime_lib
-		PREFIX ${CMAKE_CURRENT_BINARY_DIR}/runtime_libTmp
+foreach(target ${AIR_RUNTIME_TARGETS})
```

Comment:
Where is AIR_RUNTIME_TARGETS set?  The runtime didn't build at all in the github action.
I see,
```
CMake variable LibXAIE_ROOT is set to:

    /usr;/opt/xaiengine;/opt/xaienginev2
```
instead of,
```
[8/185] Performing configure step for 'air_runtime_lib'
...
-- Found LibXAIE: /home/runner/work/mlir-air/mlir-air/aienginev2/install/include  
-- Configuring done
-- Generating done
...
[1/26] Building CXX object airhost/CMakeFiles/airhost.dir/queue.cpp.o
[2/26] Building CXX object airhost/CMakeFiles/airhost.dir/memory.cpp.o
...
```